File: ControleFutebolWeb/Models/Api/JogadorDtos.cs
namespace ControleFutebolWeb.Models.Api
{
    public class JogadorResumoDto
    {
        public int Id { get; set; }
        public string Nome { get; set; } = string.Empty;
        public string NomeExibicao { get; set; } = string.Empty;
        public string Posicao { get; set; } = string.Empty;
        public int Idade { get; set; }
        public int? NumeroCamisa { get; set; }
        public string? NacionalidadeNome { get; set; }
        public int TimeId { get; set; }
        public string? TimeNome { get; set; }
        public string? FotoUrl { get; set; }
    }

    public class JogadorDetalheDto : JogadorResumoDto
    {
        public DateTime? DataNascimento { get; set; }
        public int? SelecaoId { get; set; }
        public string? SelecaoNome { get; set; }
        public string? LinkTransfermarket { get; set; }
        public string? Observacoes { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/comentarista/futebol/data/remote/dto/JogadorDtos.kt
package com.comentarista.futebol.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class JogadorResumoDto(
    val id: Int,
    val nome: String,
    val nomeExibicao: String,
    val posicao: String,
    val idade: Int,
    val numeroCamisa: Int? = null,
    val nacionalidadeNome: String? = null,
    val timeId: Int,
    val timeNome: String? = null,
    val fotoUrl: String? = null
)

@Serializable
data class JogadorDetalheDto(
    val id: Int,
    val nome: String,
    val nomeExibicao: String,
    val posicao: String,
    val idade: Int,
    val numeroCamisa: Int? = null,
    val nacionalidadeNome: String? = null,
    val timeId: Int,
    val timeNome: String? = null,
    val fotoUrl: String? = null,
    val dataNascimento: String? = null,
    val selecaoId: Int? = null,
    val selecaoNome: String? = null,
    val linkTransfermarket: String? = null,
    val observacoes: String? = null
)
